Client/Application/LeanplumIntegration.swift

```diff
@@ -196,7 +217,13 @@ class LeanPlumClient {
             Leanplum.setAppId(settings.appId, withProductionKey: settings.productionKey)
             setupType = .production
         }
-
+        
+        // To be removed after beta testing
+        // Creating a beta user id for Leanplum testing as recommedded by LP Engineers
+        let userID = UUID().uuidString + "-Beta"
+        Leanplum.setDeviceId(UIDevice.current.identifierForVendor?.uuidString)
```

Comment:
I see three IDs by this point `leanplumDeviceId`, `userDeviceID`, and now `UIDevice.current.identifierForVendor`?